tests/test_config_loader.py

```diff
@@ -446,6 +446,34 @@ def test_assign_null(
         assert cfg == {"normal_yaml_config": True, "abc": None}
         assert len(recwarn) == 0
 
+    def test_sweep_config_cache(self, path: str) -> None:
```

Comment:
That's a good start. but you need to test two things:
1. that `now` is cached.
2. that HOME environment variable is not.

I I suspect you are only testing one of the two.

Reply:
That‘s taken care of here: `assert "now" in sweep_cfg_cache.keys() and len(sweep_cfg_cache.keys()) == 1`. Nothing except the now key must be in the cache in the test.

Reply:
well, test it functionally:
monkeypatch the environment variable after you access it the first time and see that you get the new value.

Reply:
Alright, this is addressed in the latest commit